lib/client.js

```diff
@@ -250,15 +241,23 @@ class Client {
     return this.queue;
   }
 
-  formatSteps(stack, steps) {
-    return stack ? `${steps}\n\n${chalk.bold.red('################[ Failure ]################')}\n${stack}` : steps;
+  /**
+   * Returns the formatted stack including the stack trace, steps, and logs.
+   * @returns {string}
+   */
+  formatLogs({ error, steps, logs }) {
+    let testLogs = '';
+    if (steps) testLogs += `${chalk.bold.blue('Steps:')}\n${steps}`;
+    if (logs) testLogs += `\n\n${chalk.bold('Logs:')}\n${logs}`;
```

Comment:
Let's keep consistent format. If Failure is 

```
################[ Failure ]################
```
Logs should be
```
################[ Logs ]################
```

Also please check that steps and logs are not empty, for instance try to use `trim()` to check that they are not just empty string.